snap-src/bin/config

```diff
@@ -60,6 +60,10 @@ DESCRIPTION_WITH_API="Enable/disable the api of wekan"
 DEFAULT_WITH_API="true"
 KEY_WITH_API="with-api"
 
+DESCRIPTION_METEOR_SETTINGS="Meteor client settings, a JSON string, {\"public\":{}}"
```

Comment:
@whowillcare 

Oh no I accidentally merged this. Please remove this METEOR_SETTINGS JSON string completely. In general, settings are normal strings for each settings, it is set as environment variable at serverside, then it is read server side by `wekan/server/something.js` where public variable is finally set. Then at `wekan/client/` it's read from public variable.

Reply:
Hmm I think I'll already remove this myself.